src/components/Comments.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+import { getArticleComments } from '../utils/apis';
+import Accordion from 'react-bootstrap/Accordion';
+
+export default function Comments({ id }) {
+  // Does this need to be even in a state?
+  const [articleComments, setArticleComments] = useState([]);
+
+  useEffect(() => {
+    getArticleComments(id).then(({ comments }) => {
+      setArticleComments(comments);
+    });
+    //
+  }, []);
+  // console.log(articleComments);
```

Comment:
removed in future PRs

Reply:
You should consider loading state for the useEffects to better the users experience